### PR TITLE
Ts/llm thought output for release

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
# Description
Capture the llm thoughts in the input and emit them
Utilize those thoughts in a component to display them when the user hits a button/hide them when they dont

# Notes
- Not saving the llm thoughts at the moment
- all llm thoughts are replacing one another.

# Pictures
image

image

### CODE REVIEW

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<template>
+	<div v-if="showThoughts" class="thought-bubble">
+		<p>{{ thought }}</p>
+	</div>
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue:4]:
```suggestion
<p v-if="showThoughts">{{ thought }}</p>
```

Comment (shawnyama) [reply packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue:4]: Then `.thought-bubble { ...` can be `p { ...`

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const showThoughts = ref(false);
+const thought = computed(() => props?.llmThought?.content?.thought ?? '');
+</script>
+
+<style scoped>
+.thought-bubble {
+	border: 1px solid var(--surface-border-light);
+	border-radius: var(--border-radius);
+	padding: 0.75rem;
+	margin-top: 0.5rem;
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue:29]: use `var(--gap) variables

File: packages/client/hmi-client/src/components/workflow/ops/decapodes/tera-decapodes-drilldown.vue [hunk 3]
```diff
@@ -31,8 +31,10 @@
 						:defaultOptions="sampleAgentQuestions"
 						:context-language="contextLanguage"
 						@llm-output="(data: any) => appendCode(data, 'code')"
+						@llm-thought-output="(data: any) => (llmThought = data)"
 					/>
 				</Suspense>
+				<tera-notebook-jupyter-thought-output :llmThought="llmThought" />
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/decapodes/tera-decapodes-drilldown.vue:37]:
```suggestion
<tera-notebook-jupyter-thought-output :llm-thought="llmThought" />
```

File: packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue [hunk 4]
```diff
@@ -81,12 +81,16 @@
...
-				/>
+				<div class="toolbar">
+					<tera-notebook-jupyter-input
+						:kernelManager="kernelManager"
+						:defaultOptions="sampleAgentQuestions"
+						@llm-output="appendCode"
+						@llm-thought-output="(data: any) => (llmThought = data)"
+						:context-language="contextLanguage"
+					/>
+					<tera-notebook-jupyter-thought-output :llmThought="llmThought" />
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue:92]:
```suggestion
<tera-notebook-jupyter-thought-output :llm-thought="llmThought" />
```

File: packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue [hunk 5]
```diff
@@ -42,13 +42,16 @@
...
+				<div class="toolbar">
+					<tera-notebook-jupyter-input
+						:kernel-manager="kernelManager"
+						:default-options="[]"
+						:context-language="'python3'"
+						@llm-output="(data: any) => processLLMOutput(data)"
+						@llm-thought-output="(data: any) => (llmThought = data)"
+					/>
+					<tera-notebook-jupyter-thought-output :llmThought="llmThought" />
+				</div>
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/workflow/ops/stratify-mira/tera-stratify-mira.vue:54]: This pattern with the toolbar div and the components inside is repetitive. The left padding should just be default spacing. We should save this for later - just wanted to point it out.

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue (hunk 2 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/llm/tera-notebook-jupyter-thought-output.vue:29]: Thanks!

**Pull request merged**